### ISSUE TITLE
♿ Android - Accessibility - Polls - Multiple option element does not announce selection state with TalkBack

### ISSUE THREAD

**Issue by ommacc** [CMMS-Esmorga/Esmorga-Project#71]
**Original Jira Issue:** [ESM-451](https://corunamobilemakers.atlassian.net/browse/ESM-451)

## Description

**Current result: **TalkBack announces on all the options (*“Casilla de verificación. Toca dos veces para activar”*), regardless of whether the option is selected or not. The **selected state is only announced when navigating to the icon inside the option. **The entire row is clickable, therefore, the checked state should be announced when focusing the row itself.

**Expected result: **TalkBack should announce the state of each option, for example:

- *“Option A, checkbox, selected. Double tap to deactivate”*
- *“Option B, checkbox, not selected. Double tap to activate”*

This allows screen reader users to understand which options are currently selected.

## Metadata

- **Type:** Bug
- **Status:** READY FOR DEV
- **Original Jira created:** 2026-03-02T18:06:49.267+0100
- **Priority:** Medium
- **Reporter:** Mobile Makers
- **Assignee:** Android
- **Labels:** Android

## Custom Fields

- **Component:** App - Frontend
- **Environment:** QA
- **Priority:** 🔵 Low
- **Steps to reproduce:** - Enable TalkBack
- Log in
- Navigate to a poll with multiple option response
- Select more than one option
- Navigate through the options
- **Rank:** 0|i002pr:
- **Development:** {}

## Related Issues
- **relates to:** #117
